### PR TITLE
Improvement: template card

### LINKED ISSUE #256: HomePage: redesign template card
## Description

Redesign template card

Not to implements:
- black and white style

Expected design:
image

### PR DESCRIPTION
**Pull request by Rahzbob**
Resolve #256

### CODE REVIEW

**Review by justinehell — CHANGES_REQUESTED** [review]

**Comment by k-y0u** [comment]
Chrome:
image

Firefox:
image

(I scroll down at max on firefox and it can't display template name)

File: src/components/card/TemplateCard.vue [hunk 1]
```diff
@@ -4,18 +4,22 @@
...
-      height="100%"
-    >
+    <q-card-section class="q-pa-none">
+      <q-img
+        class="rounded-borders"
+        :src="`/template-library/templates/${template.key}/icon.svg`"
+      />
+    </q-card-section>
+    <q-card-section class="q-px-none q-py-sm">
       <div
```
Comment (k-y0u) [inline src/components/card/TemplateCard.vue:16]: `div` not need anymore. Move its attribute to parent `q-card-section`.

File: src/components/card/TemplateCard.vue [hunk 2]
```diff
@@ -31,6 +35,6 @@ defineProps({
 <style scoped>
 .template-card {
   width: 150px;
-  height: 150px;
+  height: 200px;
```
[the remark this answers is not in the document]
Comment (Rahzbob) [reply src/components/card/TemplateCard.vue]: done

File: src/components/card/TemplateCard.vue (hunk 1 above)
Comment (Rahzbob) [reply src/components/card/TemplateCard.vue]: done

**Review by k-y0u — APPROVED** [review]

**Review by justinehell — APPROVED** [review]

**Pull request merged**